src/main/java/com/airlines/dao/impl/AirplaneDaoImpl.java

```diff
@@ -0,0 +1,174 @@
+package com.airlines.dao.impl;
+
+import com.airlines.dao.AirplaneDao;
+import com.airlines.dao.DbConnector;
+import com.airlines.exception.DaoOperationException;
+import com.airlines.model.Airplane;
+import com.airlines.model.AirplaneModel;
+
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class AirplaneDaoImpl implements AirplaneDao {
+
+    private final static String INSERT_AIRPLANE_QUERY =
+            "INSERT INTO airplane (id, code_name, model, manufacture_date, capacity, flight_range, crew_id) " +
+                    "VALUES(?, ?, ?, ?, ?, ?, ?);";
+    private final static String SELECT_AIRPLANE_BY_ID_QUERY = "SELECT * FROM airplane WHERE id = ?;";
+    private final static String SELECT_ALL_AIRPLANES_QUERY = "SELECT * FROM airplane;";
+    private static final String DELETE_AIRPLANE_QUERY = "DELETE FROM airplane WHERE id = ?;";
+    private final static String UPDATE_CREW_ID_QUERY = "UPDATE airplane SET crew_id = ? WHERE id = ?;";
+    private final static String SELECT_AIRPLANES_BY_CREW_NAME_QUERY = """
```

Comment:
I suggest following symmetry and consistency principle:
for same cases use same approach.

In INSERT_AIRPLANE_QUERY you are using simple String concatenation through + operator

In SELECT_AIRPLANES_BY_CREW_NAME_QUERY you are using text blocks. I understand, that this piece of code is bigger, but it's still same case - some multiline String constant.

Just choose one and use it for similar case. I guess text block would be better, since you are using jdk 17

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/1277c24daafc1469479f27cad63dd700e39ab392